src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -342,6 +342,8 @@ struct MHAKernel<ScaledDotProductAttention::KT_MLAS, float> {
 
         parallel_for3d(B, H, m_blocks, [&](size_t b, size_t h, size_t m_blk) {
             size_t thread_id = static_cast<size_t>(parallel_get_thread_num());
+            if (thread_id < 0)
```

Comment:
Get `parallel_get_thread_num` as signed check if negtive (throw) and then cast?
How `size_t` can be negative?

Reply:
@praasz Try "static_cast<size_t>(-2)" with GCC13.1, get 18446744073709551614.
So I think we don't need to convert to size_t, just need to if negtive (throw) and then use it.